.redhat-ci.Dockerfile

```diff
@@ -16,7 +16,7 @@ RUN dnf install -y \
         python-dbus \
         pylint \
         python-slip-dbus \
-        python-docker-py \
+        python2-docker \
```

Comment:
But the container is based on F24 (should probably be updated to F25 now), so this isn't going to resolve, right?

Sounds instead like we should set up a rawhide tester (though of course, this issue there is that you'll have to deal with false positives).